aci/resource_aci_rtctrlattrp.go

```diff
@@ -1503,6 +1609,19 @@ func resourceAciActionRuleProfileRead(ctx context.Context, d *schema.ResourceDat
 
 	// rtctrlSetASPathASN - Read finished successfully
 
+	// rtctrlSetDamp - Beginning Read
+	setDampeningDn := dn + fmt.Sprintf("/"+models.RnrtctrlSetDamp)
+	rtctrlSetDamp, err := getRemoteRtctrlSetDamp(aciClient, setDampeningDn)
+	if err == nil {
+		log.Printf("[DEBUG] %s: rtctrlSetDamp - Beginning Read", setDampeningDn)
+		_, err = setRtctrlSetDampAttributes(rtctrlSetDamp, d)
+		if err != nil {
+			return nil
+		}
+		log.Printf("[DEBUG] %s: rtctrlSetDamp - Read finished successfully", setDampeningDn)
+	}
+	// rtctrlSetDamp - Read finished successfully
```

Comment:
You need a else condition to reset the value of set_dampening if someone unconfigure it in the UI.